module/vision/VisualMesh/data/config/docker/VisualMesh.yaml

```diff
@@ -0,0 +1,19 @@
+# The settings for each camera
+# CameraName:
+#   concurrent: How many engines to make available to concurrently execute networks
+#   engine: Which engine to use (opencl, cpu)
+#   cache_directory: directory to save compiled opencl binaries
+#   spotlight:
+#     distance: the distance range for the spotlight mesh to be generated for
+#     radius: the search radius to do at that distance (NOT THE RADIUS OF THE OBJECT)
+
+cameras:
+  Left:
+    concurrent: 2
+    network: config/networks/RobocupNetwork.yaml
+    engine: opencl
```

Comment:
hmmm if we're adding in a docker config it shouldn't be on the opencl engine. I'm not sure if it makes sense to have a docker config. if it's not nugus or webots, there shouldn't be camera data.